clang/lib/3C/AVarBoundsInfo.cpp

```diff
@@ -61,15 +61,433 @@ void AVarBoundsStats::print(llvm::raw_ostream &O,
   }
 }
 
+bool hasArray(ConstraintVariable *CK, Constraints &CS) {
```

Comment:
Unfortunately the diff in this file is not very helpful; a bunch of changes seem to just be "tons of add, tons of delete" so it's hard to tell what changed.